doc/nrf/app_build_system.rst

```diff
@@ -0,0 +1,44 @@
+.. _app_build_system:
+
+Build and configuration system
+##############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The |NCS| build and configuration system is based on the one from Zephyr, with some additions.
+
+Zephyr's build and configuration system
+***************************************
+
+Zephyr's build and configuration system uses the following building blocks as a foundation:
+
+* CMake, the cross-platform build system generator,
```

Comment:
```suggestion
* CMake, the cross-platform build system generator.
```
I know this was here before but we can use this chance to correct these small things.